backend.native/tests/stdlib_external/text/StringEncodingTestNative.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2010-2017 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package test.text
+
+
+internal actual val surrogateCodePointDecoding: String = "���"
```

Comment:
What's that? Better use \uXXX notation.

Reply:
It's a replacement characater, that looks exactly like that.

Reply:
Yeah, but I suggest to use explicit \uXXX notation.

Reply:
Fixed in PR #2973